scene/resources/packed_scene.cpp

```diff
@@ -450,7 +450,11 @@ Node *SceneState::instantiate(GenEditState p_edit_state) const {
 			callable = callable.bindp(argptrs, binds.size());
 		}
 
+#ifdef TOOLS_ENABLED
```

Comment:
Would remove this ifdef too, probably not worth it

Reply:
I don't know... CONNECT_PERSIST can technically also be used in release builds when building PackedScenes at run-time